lib/ship.rb

```diff
@@ -0,0 +1,17 @@
+class Ship
+  attr_accessor :name, :length, :health
+
+  def initialize(name,length)
+    @name = name
+    @length = length
+    @health = length
+  end
+
+  def sunk?
+    @health == 0
```

Comment:
There's a nice helper `.zero?` to make it a bit more English looking.

Also, since it's an attr_accessor, we don't need to invoke the instance variable itself.
```suggestion
    health.zero?
```

Reply:
Ah yeah! That would work too. I actually originally had `@health =< 0`, just to prevent it from returning false in the event that you call `ship.hit` again. In theory, you shouldn't hit a ship that you've already sunk, but ya know, just in case.